packages/monitoring-system/metrics-dashboard/scripts/deploy.sh

```diff
@@ -0,0 +1,17 @@
+# build and deploy the dashboard from a fresh clone
+
+set -eo pipefail  # fail-fast and don't mask errors
+
+# check token was passed in args
+if [[ $# -ne 1 ]]
+then
+  echo "Usage $0 <path-to-token-file>"
+  exit 1
+fi
+
+# compile
+npm install
+npm run compile
+
+# deploy to Firebase
+firebase deploy --token $(cat $1)
```

Comment:
Do we need to quote the token?

Reply:
I don't believe it's necessary -- I've tested it with an actual token and seems to work fine -- but I can add it anyway